web/apps/labelstudio-e2e/cypress.config.ts

```diff
@@ -2,5 +2,10 @@ import { defineConfig } from 'cypress';
 import { nxE2EPreset } from '@nx/cypress/plugins/cypress-preset';
 
 export default defineConfig({
-  e2e: nxE2EPreset(__dirname),
+    e2e: {
+        ...nxE2EPreset(__dirname),
+        // Please ensure you use `cy.origin()` when navigating between domains and remove this option.
+        // See https://docs.cypress.io/app/references/migration-guide#Changes-to-cyorigin
+        injectDocumentDomain: true
+    }
```

Comment:
indents are wrong here
```suggestion
  e2e: {
    ...nxE2EPreset(__dirname),
    // Please ensure you use `cy.origin()` when navigating between domains and remove this option.
    // See https://docs.cypress.io/app/references/migration-guide#Changes-to-cyorigin
    injectDocumentDomain: true
  }
```

Reply:
ahh does linter not catch this for us?

Reply:
I don't think it does, since I had all green on linting before it was pushed up.